src/main/resources/changelog/changelog-1.0.0.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+  <include file="changes/v1.0.0/create-currencies-table.xml" relativeToChangelogFile="true"/>
+  <include file="changes/v1.0.0/create-assigned-users-table.xml" relativeToChangelogFile="true"/>
+  <include file="changes/v1.0.0/create-tags-table.xml" relativeToChangelogFile="true"/>
+  <include file="changes/v1.0.0/create-holdings-table.xml" relativeToChangelogFile="true"/>
+  <include file="changes/v1.0.0/create-usage-consolidation-credentials-table.xml" relativeToChangelogFile="true"/>
+  <include file="changes/v1.0.0/create-usage-consolidation-settings-table.xml" relativeToChangelogFile="true"/>
+  <include file="changes/v1.0.0/create-kb-credentials-table.xml" relativeToChangelogFile="true"/>
```

Comment:
Should this one be the first one in the list? Because all other are dependent on this one?